taxi/views.py

```diff
@@ -87,3 +90,33 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
all is redundant

Reply:
What do you mean? This code was there before I added my code, I don't think we're supposed to write a code from zero, nothing in this task says otherwise.